popup/popup.js

```diff
@@ -141,7 +141,14 @@ function initPopup() {
           $('label', info).textContent = t('unreachableAMO');
           const note = (FIREFOX < 59 ? t('unreachableAMOHintOldFF') : t('unreachableAMOHint')) +
                        (FIREFOX < 60 ? '' : '\n' + t('unreachableAMOHintNewFF'));
-          const renderToken = s => s[0] === '<' ? $create('b', tWordBreak(s.slice(1, -1))) : s;
+          const renderToken = s => s[0] === '<'
+            ? $create('strong', {
```

Comment:
I think this should be changed to `code`.

BTW, I think it worth making this feature work everywhere i.e. we can `makeCopyable(element)` on any element (or on `code` elements? https://clipboardjs.com/).

Reply:
That's probably a better idea.

Reply:
Figured you passed on weighing in on this PR.

>BTW, I think it worth making this feature work everywhere

I'm scratching my head trying to figure out where else it'll be useful. If we wanna make them accessible, not sure why wouldn't we follow the regular formula. Make em links, prevent the default, tabindex of 0, and switch to `const target = document.activeElement;`.

I'd hook that up real quick if you want. Otherwise, if you guys think `clipboardjs` is useful elsewhere currently, or will be in the future, knock yourselves out.

Reply:
We're supporting modern browsers that support the clipboard API, so clipboard.js isn't necessary.

When I said it was probably a better idea, I was referring to using `<code>` blocks versus `<strong>`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API#Browser_compatibility - We shouldn't care about M$ compatibility in this case since we're only showing this content to Firefox users.

Reply:
I posted the link for an example. You can see how we can copy the sample code easily on that website.